README.md

```diff
@@ -236,479 +236,900 @@ The base class sets defaults used by other defined types, such as `tomcat::insta
+*Required if `connector_ensure` is set to `true` or `present`.* Sets a TCP port on which to create a server socket. Maps to the [port XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/http.html#Common_Attributes).
+
+Valid options: a string.
 
 #####`protocol`
 
-Specifies a protocol to use for handling incoming traffic. Maps to the [protocol XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/http.html#Common_Attributes). Valid options: a string. Default: $name.
+Specifies a protocol to use for handling incoming traffic. Maps to the [protocol XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/http.html#Common_Attributes).
+
+Valid options: a string.
+
+Default value: `$name`.
 
 #####`purge_connectors`
 
-Specifies whether to purge any unmanaged Connector elements that match defined protocol but have a different port from the configuration file. Valid options: 'true' and 'false'. Default: 'false'.
+Specifies whether to purge any unmanaged Connector elements that match defined protocol but have a different port from the configuration file.
+
+Valid options: `true` and `false`.
+
+Default value: `false`.
 
 #####`server_config`
 
-Specifies a server.xml file to manage. Valid options: a string containing an absolute path. Default: '${catalina_base}/config/server.xml'.
+Specifies a server.xml file to manage.
+
+Valid options: a string containing an absolute path.
+
+Default value: `${catalina_base}/config/server.xml`.
 
 ####tomcat::config::server::context
 
 #####`additional_attributes`
 
-Specifies any further attributes to add to the Context. Valid options: a hash of '< attribute >' => '< value >' pairs. Default: {}.
+Specifies any further attributes to add to the Context.
+
+Valid options: a hash of '< attribute >' => '< value >' pairs.
+
+Default value: `{}`.
 
 #####`attributes_to_remove`
 
-Specifies any attributes to remove from the Context.  Valid options: a hash of '< attribute >' => '< value >' pairs. Default: [].
+Specifies any attributes to remove from the Context. 
+
+Valid options: a hash of '< attribute >' => '< value >' pairs.
+
+Default value: `[]`.
 
 #####`catalina_base`
 
-Specifies the base directory of the Tomcat installation to manage. Valid options: a string containing an absolute path. Default: $::tomcat/catalina_home.
+Specifies the base directory of the Tomcat installation to manage.
+
+Valid options: a string containing an absolute path.
+
+Default value: `$::tomcat/catalina_home`.
 
 #####`context_ensure`
 
-Specifies whether the [Context XML element](http://tomcat.apache.org/tomcat-8.0-doc/config/context.html) should exist in the configuration file. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Specifies whether the [Context XML element](http://tomcat.apache.org/tomcat-8.0-doc/config/context.html) should exist in the configuration file.
+
+Valid options: `true`, `false`, `present`, and `absent`.
```

Comment:
true and false are correct in backticks, but present and absent are strings, rather than true Booleans, so should be in single quotes.